lexicons/tools/ozone/moderation/queryStatuses.json

```diff
@@ -28,6 +28,31 @@
             "format": "datetime",
             "description": "Search subjects reviewed after a given timestamp"
           },
+          "recordDeletedAfter": {
```

Comment:
My qualm with these fields is just that they are named "record" when they really may apply to any subject, e.g. an account rather than a record.